src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2921,6 +2921,11 @@ void processContextElement
         buildGeneralErrorResponse(ceP, NULL, responseP, SccInvalidModification, "Already Exists");
         return;
     }
+    else if ((checkEntityExistance == true) && (action == "APPEND") && (entitiesNumber == 0))
```

Comment:
Why don't just `checkEntityExistance` instead of `checkEntityExistance == true`? ;)